libraries/TFT_Touch_Shield_V2/TFTv2.cpp

```diff
@@ -204,10 +204,10 @@ void TFT::fillScreen(INT16U XL, INT16U XR, INT16U YU, INT16U YD, INT16U color)
         YD = YU^YD;
         YU = YU^YD;
     }
-    XL = constrain(XL, MIN_X,MAX_X);
-    XR = constrain(XR, MIN_X,MAX_X);
-    YU = constrain(YU, MIN_Y,MAX_Y);
-    YD = constrain(YD, MIN_Y,MAX_Y);
+    XL = (XL > MAX_X) ? MAX_X : XL;
```

Comment:
how about using std::min here?

Reply:
By the way, why is constraint on MIN_Y being dropped?

Reply:
GCC is pedantic here.  MIN_X/Y =0, and constrain is a macro that compares a to b  So you end up with "(uint16_t)XR < 0" as one of the conditions it checks, and that's by definition always false.  I will put std::min in on next push.